pkg/manifestclient/list.go

```diff
@@ -74,6 +75,13 @@ func (mrt *manifestRoundTripper) listAll(requestInfo *apirequest.RequestInfo) ([
 		if retList.GroupVersionKind() != kind.listKind {
```

Comment:
I know that a label selector can also be specified for watch requests. 
Would it be worth adding a simple test for this case to ensure it won't blow up?

The current implementation doesn't support watching changes on resources.

Reply:
I'm not sure I'm following, but can this be done when we support Watch?

Reply:
We will never support watch.  Watch will always just hold a connection open so that we never encourage live mutation and thus allow interdependence between loops.

Reply:
We discussed on Slack and agreed that a test for List should be enough, since we don't support Watch. @deads2k let me know if you disagree.